spec/kong_tests.conf

```diff
@@ -8,6 +8,9 @@ cluster_listen_rpc = 127.0.0.1:9373
 ssl_cert = spec/fixtures/kong_spec.crt
 ssl_cert_key = spec/fixtures/kong_spec.key
 
+admin_ssl_cert = spec/fixtures/kong_spec.crt
+admin_ssl_cert_key = spec/fixtures/kong_spec.key
```

Comment:
I think we should disable `admin_ssl` (and `ssl` altogether) from being used in our tests. We do not have any test even making use of them, and if that is or becomes the case, we can enable SSL for this particular test when starting kong in its `setup()` hook.

We should keep the test instance as lean as possible.